buildconfig/CMake/LinuxPackageScripts.cmake

```diff
@@ -214,11 +239,20 @@ execute_process(COMMAND "chmod" "+x" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/AddPytho
 # Package version
 unset(PYTHON_ARGS)
 
-# used by mantidplot and mantidworkbench
-set(LOCAL_PYPATH "\${INSTALLDIR}/bin:\${INSTALLDIR}/lib:\${INSTALLDIR}/plugins")
+# used by mantidworkbench
+if(CONDA_BUILD)
```

Comment:
I just need to check how we are launching workbench on our analysis machines as I'm not sure they have `conda run` before them.

Reply:
We are activating the conda environment before running the `mantidworkbench` script.

It is moderately convoluted. I didn't know that `conda run` was a thing. Thanks for the tip!